SQL target keywords (#833)
Codecademy/docs: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md

---
Title: 'FULL OUTER JOIN'
Description: 'Combines rows from different tables even if the JOIN condition is not met.'
Subjects:
  - 'Data Science'
Tags:
  - 'Database'
  - 'Queries'
  - 'Join'
  - 'PostgreSQL'
  - 'MySQL'
  - 'SQLite'
  - 'Tables'
CatalogContent:
  - 'learn-sql'
  - 'paths/analyze-data-with-sql'
  - 'paths/design-databases-with-postgresql'
---

> The **`FULL OUTER JOIN`** command combines matching rows with rows from both the left- and right-side tables.

Reviewer: ```suggestion
The **`FULL OUTER JOIN`** command combines matching rows with all rows from both the left- and right-side tables.
```

## Syntax

```pseudo
SELECT column(s)
FROM table_A
FULL OUTER JOIN table_B

> ON table_A.column(s) = table_B.column(s)

Reviewer 2: ```suggestion
  ON table_A.column(s) = table_B.column(s)
```

WHERE condition;
```

> One or more matching `column(s)` can be selected and joined from `table_A` and `table_B` based on matching columns between tables in the `ON` clause. The results can be further filtered based on a `condition` in the [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where) clause.

Reviewer: ```suggestion
One or more matching `column(s)` can be selected and joined from `table_A` and `table_B` based on matching columns between tables in the `ON` clause to define which rows are combined. Other rows from each table appear in the result separately. The results can be further filtered based on a `condition` in the [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where) clause.
```

## Example

> The following example selects the all rows (`*`) from the tables `books` and `authors` that have matching author IDs less than 11:

Reviewer: ```suggestion
The following example selects the all rows (`*`) from the table `books` with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:
```

> The following example selects the all rows (`*`) from the table `books` with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:

Reviewer 2: ```suggestion
The following example selects all the rows (`*`) from the `books` table with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:
```
